### DISCUSSION TITLE
Brkraw uses the Study ID instead of Session ID

### DISCUSSION THREAD

**Discussion by arefks** [BrkRaw/brkraw#172] [General]
Hi to all, 

I am curious about why, recently, when I convert my raw data, brkraw uses the Study ID in the naming of the NIfTI files. In older commits, brkraw used the Session ID. Has anyone else experienced something similar or noticed any changes in the standards? Or was this change intended? Is this a change in the standard, or was it a bug before and now it is the correct version? Additionally, is this something that can be set by the user?

![EWHKFBARnAeIShGlWJkgxoyx](https://github.com/BrkRaw/brkraw/assets/87692940/1a66b456-c02e-4b3a-b93c-1fb211556b53)
The upper folder is from today and has used ses-1 (Study ID).
The lower folder is from older commits and used ses-Baseline (Session ID).

**Comment by dvm-shlee** [comment]
Hi, @arefks Sorry for super late on responding on this topic.

Good question. Of course, BIDS conventions are important, but in BrkRaw we generally try to respect the original Paravision terminology as much as possible when parsing metadata. My understanding is that this change - using the Study ID instead of the Session ID - was introduced from that perspective, rather than being driven by a strict BIDS rule change. I do not think this was meant as a correction of a bug, but rather an intentional design choice at that time.

In the recent updates, we did not further modify this convention directly. However, with the new release, we introduced brkraw-viewer as part of the updated workflow.
- [brkraw-viewer](https://github.com/brkraw/brkraw-viewer.git) is now separated from the core package and provided as a CLI extension:

- After installing brkraw==0.5.0rc1, you can install brkraw-viewer from GitHub and use it via:
```
brkraw viewer <path>
```
- The viewer provides additional options for reorientation and customization of output filenames, allowing users to adapt the layout more flexibly without changing the core conversion logic.
- For details on layout rules and filename customization, please refer to the documentation here:
https://brkraw.github.io/reference/layout.html
